Add `load_dot_env()` function for loading the context's environment from a dotenv file (#101)
jolicode/castor: doc/05-context.md

    run('pwd'); // will print /tmp
}
```

## Load a context from dotenv file

> You can also load your context from a dotenv file:

Reviewer: ```suggestion
You can also load your context from a dotenv file by calling `Context::fromDotEnv()`:
```

```php
use Castor\Attribute\AsContext;
use Castor\Context;

#[AsContext]
function my_context(): Context
{
    return new Context::fromDotEnv();
}
```

> By default, it loads the `.env` file on your project root (where castor file or
> folder was found), but you can overload this by passing your dotenv file

Reviewer: ```suggestion
By default, it loads the `.env` file on your project root (where your `castor.php` file
was found), but you can overload this by passing your dotenv file
```

> path as an argument.

Reviewer: ```suggestion
path as the first argument of the method `Context::fromDotEnv()`.
```
